src/components/hub/ChatForm.js

```diff
@@ -57,6 +58,7 @@ export default function ChatForm() {
     dispatch(setClicked(false));
     dispatch(clearSelectedConversation());
     api.unsubscribeFromUserActivity({});
+    messagesService.clearPrevConvMessagesToLocalLimit(selectedCID);
```

Comment:
do we also do this cleaning when switch between convs?

Reply:
vvv
https://github.com/SAMA-Communications/sama-client/pull/135/files#diff-f13243307d8bf1149546c0e0e90c863503bece4d3d008d43bf5a4caec63556f6R164